idd/Energy+.idd.in

```diff
@@ -23577,18 +23595,20 @@ OutputControl:IlluminanceMap:Style,
 \group Zone Airflow
 
 ZoneInfiltration:DesignFlowRate,
-       \memo  Infiltration is specified as a design level which is modified by a Schedule fraction, temperature difference and wind speed:
-       \memo  Infiltration=Idesign * FSchedule * (A + B*|(Tzone-Todb)| + C*WindSpd + D * WindSpd**2)
-       \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
-       \memo to all the zones in the ZoneList.
+       \memo Infiltration is specified as a design level which is modified by a Schedule fraction, temperature difference and wind speed:
+       \memo Infiltration=Idesign * FSchedule * (A + B*|(Tzone-Todb)| + C*WindSpd + D * WindSpd**2)
+       \memo If a ZoneList, SpaceList, or a Zone comprised of more than one Space is specified
+       \memo then this definition applies to all applicable spaces, and each instance will
+       \memo be named with the Space Name plus this Object Name.
```

Comment:
I'll have to keep this in mind when reviewing. It sounds like if a Zone name is used, and that zone has more than 1 space then a ZoneInfiltration:DesignFlowRate object will be created for each space names SpaceNum + ObjectName. What I want to look for is if the input also includes a ZoneInfiltration:DesignFlowRate for each of the zone spaces (i.e., user already defined the ZoneInfiltration:DesignFlowRate objects) then will there be only 1 ZoneInfiltration:DesignFlowRate object for that space, or two.